src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -37,9 +37,17 @@ import "./ScheduleItemNG.scss";
 
 export interface ScheduleItemNGProps {
   event: ScheduledVenueEvent;
+  isShowImage: boolean;
+  isShowBookmark: boolean;
+  isShowDayDate: boolean;
```

Comment:
It seems from current usages that these new properties either are all `true` or all `false`. Can be simplified to just one property then.